### PR TITLE
Website resource

### PR DESCRIPTION
**Pull request by deeheber**
- [x] Update stack to utilize  the website resource
- [x] Remove resources that are no longer needed
- [x] Update NPM dependencies
- [x] Update any README/Documentation

Going to merge this in with https://github.com/stackery/stackery-docs/pull/199

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -1,52 +1,164 @@
...
+            SourceDirectory: src/site
+            DestinationBucketName: !Ref FrontEnd
+      Tags:
+        - Key: Stackery Project Type
+          Value: Website Builder
+  WebsiteEvents:
+    Type: AWS::Events::Rule
+    DependsOn: Website
+    Properties:
+      Name: !Sub ${SourceVersion}-stackerywebsite
```
Comment (txase) [inline template.yaml]: Remove this line in keeping with the website resource updates :)

**Review by txase — COMMENTED** [review]
Looks good so far!

File: template.yaml (hunk 1 above)
Comment (deeheber) [reply template.yaml]: Thanks!

**Review by bildungsroman — APPROVED** [review]
🙌

**Review by JMFritz — APPROVED** [review]

**Pull request merged**